SingularityClient/src/main/java/com/hubspot/singularity/client/SingularityClient.java

```diff
@@ -408,6 +410,11 @@ public SingularityClient(
         .build();
   }
 
+  @Deprecated
+  public void setSkipApiCache(boolean skipApiCache) {
+    this.skipApiCache = skipApiCache;
+  }
```

Comment:
for structure/usability of the client. Is there a reason you have this at the class level instead of just being an arg on the relevant method(s)?

Reply:
I looked through the request call usages in Orion and talked to Suruu and we decided it would be easier on Deploy's side to have a class level argument rather than updating all usages. It also makes clean up easier for when we've solved the underlying CuratorFramework issue